README.adoc

```diff
@@ -1,3 +1,41 @@
+= C++ example project scanned on SonarCloud using Azure Pipelines
+// URIs:
+:uri-qg-status: https://sonarcloud.io/dashboard?id=sc-sq-c-family-examples_linux-cmake-azure-sc
+:img-qg-status: https://sonarcloud.io/api/project_badges/measure?project=sc-sq-c-family-examples_linux-cmake-azure-sc&metric=alert_status
+:uri-build-status: https://dev.azure.com/sc-sq-c-family-examples/linux-cmake-azure-sc/_build/latest?definitionId=2&branchName=azure-sc-analysis
+:img-build-status: https://dev.azure.com/sc-sq-c-family-examples/linux-cmake-azure-sc/_apis/build/status/sc-sq-c-family-examples.linux-cmake-azure-sc?branchName=azure-sc-analysis
+
+image:{img-build-status}[Build Status, link={uri-build-status}]
+image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]
+
+*This project is analysed on https://sonarcloud.io/dashboard?id=sc-sq-c-family-examples_linux-cmake-azure-sc[SonarCloud]!*
+
+It is very easy to analyze a C/C++/Objective-C project with SonarCloud:
+
+. Create a `sonar-project.properties` file to store your configuration
```

Comment:
All these steps are nice but are not self-sufficient (eg: how do I fill in the SONAR_TOKEN ?)
In fact, they are a summary of what is described here: https://sonarcloud.io/documentation/getting-started/azure-devops/
So I think it would be better to better highlight the link to this documentation (by putting it at the beginning for example)
WDYT?

Reply:
Agree. I added the link to the line 13.